### PR TITLE
Both `uuid?` and `semver?` should only check strs

### PR DESCRIPTION
**Pull request by acron0**
Otherwise we leak exceptions

### CODE REVIEW

File: src/kixi/comms/schema.clj [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
   [s]
-  (re-find #"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$" s))
+  (and (string? s)
+       (re-find #"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$" s)))
 
 (defn semver?
   [s]
-  (re-find #"^\d+\.\d+\.\d+$" s))
+  (and (string? s)
+       (re-find #"^\d+\.\d+\.\d+$" s)))
```
Comment (otfrom) [inline src/kixi/comms/schema.clj]: doesn't handle -alpha, -beta or -SNAPSHOT

**Comment by otfrom** [comment]
Even a 1 or 2 example tests would be good pls.

File: src/kixi/comms/schema.clj (hunk 1 above)
Comment (acron0) [inline src/kixi/comms/schema.clj]: By design! We don't expect to have alpha, beta or snapshot commands or events.

Comment (otfrom) [inline src/kixi/comms/schema.clj]: fair enough. :-)

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**